packages/round-manager/src/features/api/__tests__/utils.test.ts

```diff
@@ -97,7 +97,7 @@ describe("graphql_fetch", () => {
       }
     `
 
-    const res = await graphql_fetch(query, "goerli")
```

Comment:
I'd like to see some way of maintaining the information provided by the strings here; "goerli" means something to our domain, but "5" does not

Reply:
Yeah, I definitely agree. For now I added constants in the test file only, but it would be nice to have these in a global constants file in the codebase, since the chain IDs are also referenced in other places like `switch (chainId) { ... }`